roles/provision-metrics-apb/templates/grafana-data-sources.yml.j2

```diff
@@ -5,7 +5,7 @@ datasources:
   # <string, required> access mode. direct or proxy. Required
   access: proxy
   # <string> url
-  url: http://localhost:9090
+  url: http://prometheus-internal
```

Comment:
is it this URL because of the service name?

Reply:
Yes. We needed to use service name. 
This service will be always in the same namespace so we can use shorter form.